eya_def_tools/eya_def_tools/data_models/energy_assessment.py

```diff
@@ -12,31 +12,34 @@
 from eya_def_tools.data_models.wind_uncertainty import WindUncertaintyAssessment
 
 
-class EnergyAssessmentResults(EyaDefBaseModel):
-    """Energy assessment results."""
+class GrossEnergyAssessmentResults(EyaDefBaseModel):
```

Comment:
The data models for the gross and net energy assessment results are now separated, as the requirements and descriptions need to be somewhat different. The new requirements are as agreed during a previous schema review meeting.